x-pack/legacy/plugins/ml/public/components/navigation_menu/tabs.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useState } from 'react';
+import { EuiTabs, EuiTab } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import chrome from 'ui/chrome';
+
+interface Tab {
+  id: string;
+  name: any;
+  disabled: boolean;
+}
+
+interface TestSubjMap {
+  [key: string]: string;
+}
+
+interface Props {
+  disableLinks: boolean;
+  tabId: string;
+}
+
+function getTabs(disableLinks: boolean): Tab[] {
+  return [
+    {
+      id: 'jobs',
+      name: i18n.translate('xpack.ml.navMenu.jobManagementTabLinkText', {
+        defaultMessage: 'Job Management',
+      }),
+      disabled: disableLinks,
+    },
+    {
+      id: 'explorer',
+      name: i18n.translate('xpack.ml.navMenu.anomalyExplorerTabLinkText', {
+        defaultMessage: 'Anomaly Explorer',
+      }),
+      disabled: disableLinks,
+    },
+    {
+      id: 'timeseriesexplorer',
+      name: i18n.translate('xpack.ml.navMenu.singleMetricViewerTabLinkText', {
+        defaultMessage: 'Single Metric Viewer',
+      }),
+      disabled: disableLinks,
+    },
+    {
+      id: 'data_frames',
```

Comment:
This ID needs to match the one used in the data frames `route.ts` as otherwise the menu item isn't selected when the data frames page is displayed. I think `data_frames` with the `s` is best, as it matches the text in the menu.

Reply:
Fixed in c50fc5011d45bb260a087bb58692abd20fb4cd9c